arch/inst/F/fabs.s.yaml

```diff
@@ -0,0 +1,25 @@
+# yaml-language-server: $schema=../../../schemas/inst_schema.json
+
+fabs.s:
```

Comment:
fabs.s is a pseudoinstruction. In the UDB scheme, it should be listed as such under fsgnjx.s like so:

``` yaml
fsgnjx.s:
  # ...
  pseduoinstructions:
  - when: "fs1 == fs2"
    to: fabs.s fd, fs1
```

Reply:
Surely! The script does this by itself, I believe the reason for the error here is that I forgot to update riscv-opcodes to its most recent head. In that note, may I also commit the submodule update to it's latest commit?

Reply:
This is now solved in the most recent commit.